mixins/menu-overlay.html

```diff
@@ -0,0 +1,62 @@
+<link rel="import" href="../color.html">
+<link rel="import" href="overlay.html">
+
+<dom-module id="material-menu-overlay" theme-for="vaadin-dropdown-menu-overlay vaadin-combo-box-overlay vaadin-context-menu-overlay">
```

Comment:
Should not specify `theme-for` for this module. The individual modules should opt-in to include this module (like they actually are right now, which probably causes them to be loaded twice).